contracts/cw-proposal-single/src/contract.rs

```diff
@@ -7,9 +7,11 @@ use cosmwasm_std::{
 use cw2::set_contract_version;
 use cw_storage_plus::Bound;
 use cw_utils::{Duration, Expiration};
+use proposal_hooks::{new_proposal_hooks, new_vote_hooks, proposal_status_changed_hooks};
 
 use voting::{Vote, Votes};
 
+use crate::state::HOOKS;
 use crate::{
     error::ContractError,
```

Comment:
This is total bikeshedding but we can combine these lol
```suggestion
use crate::{
    state::HOOKS,
    error::ContractError,
```